mrbelvedereci/users/management/commands/usertoken.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
wait. i already reviewed this. is this command in here twice?

Reply:
There are 3 commands in this PR, 1 in build, 2 in users